interface/wx/notifmsg.h

```diff
@@ -64,6 +67,15 @@ class wxNotificationMessage : public wxEvtHandler
         Timeout_Never = 0   ///< Notification will never time out.
     };
 
+    /// Reasons for dismissal, posted with wxEVT_NOTIFICATION_MESSAGE_DISMISSED events.
```

Comment:
Although trivial, we probably still want to do this for consistency:

```suggestion
    /**
      Reasons for dismissal, posted with wxEVT_NOTIFICATION_MESSAGE_DISMISSED events.
      
      @since 3.2.5
    */
```